### DISCUSSION TITLE
system errors in multi chat show up as most recent in in-game chat

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#21433] [Q&A] [answered]
system errors in multi chat show up as most recent in in-game chat.
also, chat is not available before the map, as it is in breaks. it probably should be.

https://user-images.githubusercontent.com/78441708/204110244-4b35a237-c6c6-4d78-86ae-2ec28b9848d6.mp4

**Comment by Axilotl17** [comment, marked as answer]
i dont think this is a huge fix ._.

**Reply by ghost** [reply]
fixed in #21463, tracking at #21447
